modules/manually-create-identity-access-management.adoc

```diff
@@ -6,24 +6,51 @@
 
 ifeval::["{context}" == "manually-creating-iam-aws"]
 :aws:
+:cco-multi-mode:
 endif::[]
 ifeval::["{context}" == "manually-creating-iam-azure"]
 :azure:
+:cco-multi-mode:
 endif::[]
 ifeval::["{context}" == "manually-creating-iam-gcp"]
 :google-cloud-platform:
+:cco-multi-mode:
 endif::[]
+ifeval::["{context}" == "installing-azure-stack-hub-default"]
+:ash:
+:cco-manual-mode:
+endif::[]
+
 
 [id="manually-create-iam_{context}"]
+//For providers that support multiple modes of operation
+ifdef::cco-multi-mode[]
 = Manually create IAM
+endif::cco-multi-mode[]
 
+//For providers who only support manual mode
+ifdef::cco-manual-mode[]
+= Manually manage cloud credentials
+endif::cco-manual-mode[]
+
+//For providers that support multiple modes of operation
+ifdef::cco-multi-mode[]
 The Cloud Credential Operator (CCO) can be put into manual mode prior to
 installation in environments where the cloud identity and access management
 (IAM) APIs are not reachable, or the administrator prefers not to store an
 administrator-level credential secret in the cluster `kube-system` namespace.
+endif::cco-multi-mode[]
+
+//For providers who only support manual mode
+ifdef::cco-manual-mode[]
+Installing the cluster requires that the Cloud Credential Operator (CCO) operate in manual mode. While the installer configures the CCO for manual mode, you must specify the identity and access management secrets for you cloud provider.
```

Comment:
I think this wording a little confusing. The first sentence, with the use of "requires" sounds like it needs something from the user. And the second sentence it is not really the installer AFAIK that configures it, but that is not an important detail. I'm not sure what the point of this section is. The important point is that users must provide those IAM secrets. 

So perhaps we could say something like:

The Cloud Credential Operator only supports your cloud provider. in Manual Credential Mode; therefore you must specify the identity and access-management (IAM) secrets for your cloud provider.